pubspec.yaml

```diff
@@ -50,7 +50,7 @@ dependencies:
   http: ^1.2.1
   image_cropper: ^5.0.1
   image_picker: ^1.0.6
-  intl: ^0.18.0
+  intl: ^0.18.1
```

Comment:
upgraded this package because there were conflicts with older version while running `pub get`.